lib/libvfio-user.c

```diff
@@ -1162,34 +1162,18 @@ vfu_realize_ctx(vfu_ctx_t *vfu_ctx)
 }
 
 int
-vfu_ctx_drive(vfu_ctx_t *vfu_ctx)
+vfu_ctx_poll(vfu_ctx_t *vfu_ctx)
 {
     int err;
+    int blocking = vfu_ctx->flags & LIBVFIO_USER_FLAG_ATTACH_NB ? 0 : 1;
 
-    if (vfu_ctx == NULL || !vfu_ctx->realized) {
+    if (!vfu_ctx->realized) {
```

Comment:
I think we should have at least `assert(vfu_ctx != NULL)`.

Reply:
Ack